src/hash_collision_bench.cc

```diff
@@ -27,9 +27,10 @@ int random(int low, int high) {
 
 void RandomCommand(char** s) {
   int len = random(5, 100);
-  *s = new char[len];
+  *s = new char[len+1];
```

Comment:
Can you make a stand-alone PR for this?

Reply:
Sure, no problem. I debated it originally.

Reply:
Opened #1880